content/millennium/r4/diagnostic/observation.md

```diff
@@ -219,13 +219,74 @@ The `ETag` response header indicates the current `If-Match` version to use on a
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Update
```

Comment:
@Manjushavrk @BS062721 I noticed that in the 'create' section we still have the Data Absent Reason field, which shouldn't be there on creates. I think Manjusha removed it from updates. Can we make sure it is gone for creates too?

Reply:
+1

Reply:
I have removed dataAbsentReason from creates. We only have dataAbsentReason in terminology bindings. 
https://github.com/cerner/fhir.cerner.com/blob/24cb43e7f384892b8a8dbb1aaf51ec9703f2b949/lib/resources/r4/observation.yaml#L618